integration-tests/helpers/index.js

```diff
@@ -113,7 +114,10 @@ function assertTelemetryPoints (pid, msgs, expectedTelemetryPoints) {
       runtime_name: 'nodejs',
       runtime_version: process.versions.node,
       tracer_version: require('../../package.json').version,
-      pid: Number(pid)
+      pid: Number(pid),
+      result: telemetryModule.result,
```

Comment:
It seems weird that these three new fields are camelCase while all the existing fields are snake_case. Does the backend expect this disparity?